### PR TITLE
Add test for updateLanguage mutation

### LINKED ISSUE #365: [Child Issue] Test Cases for updateLanguage Mutation
Is your feature request related to a problem? Please describe.
Test Case is required for updateLanguage Mutation

Describe the solution you'd like
As a user changes his language preference into the system, we take the language input selected by the user, stores user language preference in the database and returns the user data. When writing the test code please cover all the business logic involved in the process. Go through the login Mutation for understanding the logic.

**Additional context**
Parent Issue: 
- #361

### PR DESCRIPTION
**Pull request by Prathamesh010**
**What kind of change does this PR introduce?**
Adds tests for updateLanguage Mutation.

**Issue Number:**
Fixes #365 

**Does this PR introduce a breaking change?**
No.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
Yes.

### CODE REVIEW

**Comment by palisadoes** [comment]
@Prathamesh010  The code base has been updated with a more recent pull request that should have fixed the issue. Please merge your branch with the lastest `upstream/develop` and push to your repo. GitHub should detect the changes and re-run the tests on the updated code.

**Comment by palisadoes** [comment]
@Prathamesh010 All tests passed! Let's wait on the review from @sumitra19jha

File: tests/language.spec.js
Comment (sumitra19jha) [inline tests/language.spec.js:17]: Please create an array variable containing language code, then from that array randomly select one language code and use that variable as an input here

**Review by sumitra19jha — CHANGES_REQUESTED** [review]

**Review by sumitra19jha — COMMENTED** [review]
Looks good now. Please add a snapshot where this test code is passing in the current repo

**Comment by Prathamesh010** [comment]
image

**Review by sumitra19jha — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**
